unit_tests/ut_h_rl.amlg

```diff
@@ -312,9 +320,7 @@
 					(list 0 2.5 2.7 "medium" "strawberry")
 					(list 0 1.1 .7 "small" "strawberry")
 					(list 0 1.1 .8 "small" "strawberry")
-				)
-				ablatement_params (assoc "fruit" (list "exact"))
-			))
+				)			))
```

Comment:
the closing two parens should be on the next line, matching the indent of the opening (call_entity parenthesis